### PR TITLE
Create README.md

### LINKED ISSUE #11: Create a more detailed draft documentation for the schema
Perhaps add a README.md to in the new `/schema` folder.

### PR DESCRIPTION
**Pull request by santosomar**
Fixes #11 

Creating a more detailed draft documentation for the schema

### CODE REVIEW

File: schema/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Explanation of Schema Properties
+
+1. **$schema**: This property references the schema definition (`schema_t`) and specifies the schema against which the JSON object must be validated. The valid value for this property is `"https://openplf.org/eol-eos-schema-0.2.0.json"`.
+
+2. **creationTimestamp**: This property represents the timestamp when the record was created. The timestamp must follow the RFC 3339 standard for date-time formatting. For example: `"2023-05-01T12:00:00Z"`.
+
+3. **EOLDate**: This property contains the End of Life (EOL) date of the product, indicating the date after which the product is no longer produced or maintained. The value can either be a date-time format following the RFC 3339 standard or the string "tba" (To Be Announced) if the EOL date is not yet determined.
+
+4. **supplierId**: This property contains a unique identifier for the supplier or service provider. It is a string with a minimum length of 1 character.
+
```
Comment (tschmidtb51) [inline schema/README.md:26]:
```suggestion
*Note: The exact usage and proper identifying method of this field is currently under discussion in the community (see #6). As a [CSAF](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html) issuing party, you could use the value `/document/publisher/namespace`.*

```

File: schema/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+5. **supplierName**: This property contains the name of the supplier or service provider. It is a string with a minimum length of 1 character.
+
+6. **productId**: This property contains the unique product identifier for the product. It is a string with a minimum length of 1 character and references the `productId_t` definition.
+
```
Comment (tschmidtb51) [inline schema/README.md:30]:
```suggestion
*Note: The exact usage and proper identifying method of this field is currently under discussion in the community (see #22). If you have an SBOM that you provide for download at a specific link and that one is stable, the download URL could be used as identifier.*

```

File: schema/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+7. **productName**: This property contains the name of the product. It is a string with a minimum length of 1 character and references the `productName_t` definition.
+
+8. **productVersion**: This property contains the version or release information of the product. It is a string with a minimum length of 1 character and references the `productVersion_t` definition.
+
+9. **replacementProduct**: This property contains information about the recommended replacement product, if applicable. It is an object with the following required properties:
+
+    - **productId**: Contains the unique product identifier for the replacement product. It is a string with a minimum length of 1 character and references the `productId_t` definition.
+
```
Comment (tschmidtb51) [inline schema/README.md:38]:
```suggestion
    *Note: The exact usage and proper identifying method of this field is currently under discussion in the community (see #22). If you have an SBOM that you provide for download at a specific link and that one is stable, the download URL could be used as identifier.*

```

File: schema/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+4. **supplierId**: This property contains a unique identifier for the supplier or service provider. It is a string with a minimum length of 1 character.
+
+5. **supplierName**: This property contains the name of the supplier or service provider. It is a string with a minimum length of 1 character.
+
+6. **productId**: This property contains the unique product identifier for the product. It is a string with a minimum length of 1 character and references the `productId_t` definition.
+
+7. **productName**: This property contains the name of the product. It is a string with a minimum length of 1 character and references the `productName_t` definition.
+
+8. **productVersion**: This property contains the version or release information of the product. It is a string with a minimum length of 1 character and references the `productVersion_t` definition.
```
Comment (tschmidtb51) [inline schema/README.md:33]:
```suggestion
8. **productVersion**: This property contains the version or release information of the product. It is a string with a minimum length of 1 character and references the `productVersion_t` definition. This SHALL not contain a version range.

File: schema/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+9. **replacementProduct**: This property contains information about the recommended replacement product, if applicable. It is an object with the following required properties:
+
+    - **productId**: Contains the unique product identifier for the replacement product. It is a string with a minimum length of 1 character and references the `productId_t` definition.
+    
+    - **productName**: Contains the name of the replacement product. It is a string with a minimum length of 1 character and references the `productName_t` definition.
+    
+    - **productVersion**: Contains the version or release information of the replacement product. It is a string with a minimum length of 1 character and references the `productVersion_t` definition.
```
Comment (tschmidtb51) [inline schema/README.md:41]:
```suggestion
    - **productVersion**: Contains the version or release information of the product. It is a string with a minimum length of 1 character and references the `productVersion_t` definition. This SHALL not contain a version range.

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
Some minor comments

**Comment by santosomar** [comment]
Thank you @tschmidtb51 ! I incorporated your suggested changes.

**Pull request merged**